### PR TITLE
improvement: CLDSRV-76 release 8.3.3

### PR DESCRIPTION
**Pull request by welansari**
Release 8.3.3

### CODE REVIEW

File: docs/RELEASE.md [hunk 1]
```diff
@@ -26,18 +26,55 @@ docker pull registry.scality.com/cloudserver/cloudserver:<tag>
...
+
+* With the following parameters, [force a build here](https://eve.devsca.com/github/scality/cloudserver/#/builders/3/force/force)
+
+    * Branch Name: The one used for the tag earlier. In this example 'development/8.3'
+    * Override Stage: 'release'
+    * Extra properties:
+      * name: `'tag'`, value: `[release version]`, in this example`'8.3.0'`
+
+* Once the docker image is present on [registry.scality.com](registry.scality.com),
+  update Vault's `package.json`
```
Comment (ghost) [inline docs/RELEASE.md]: It's cloudserver here! :smile:

**Review by ghost — CHANGES_REQUESTED** [review]
Minor change! Also, to ease the reviews, you can do multiple commits per PR with one logical change at a time, e.g., one commit to update the release documentation, and one other to update the version in package.json.

File: docs/RELEASE.md (hunk 1 above)
Comment (welansari) [reply docs/RELEASE.md]: Yep, great idea! thanks

**Comment by welansari** [comment]
ping

File: docs/RELEASE.md (hunk 1 above)
Comment (ghost) [reply docs/RELEASE.md]: You forgot to change `Vault` here :slightly_smiling_face:

**Review by ghost — CHANGES_REQUESTED** [review]
:arrow_down:

**Review by XinLi0207 — APPROVED** [review]
LGTM! good job

**Review by ghost — APPROVED** [review]

**Pull request merged**